tests/smrf_test_case.py

```diff
@@ -117,6 +117,11 @@ def setUpClass(cls):
         # read in the base configuration
         cls.base_config = get_user_config(config_file, modules='smrf')
 
+        # create the output dir
+        folder = os.path.join(cls.base_config.cfg['output']['out_location'])
+        if not os.path.exists(folder):
+            os.mkdir(folder)
```

Comment:
There is a call within Python >= 3.2 to make this one line:
https://docs.python.org/3/library/os.html#os.makedirs
```python
os.makedirs(folder, exist_ok=True)
```

Reply:
Awesome!